### PR TITLE
Add Email Preview API documentation

### PR DESCRIPTION
**Pull request by jeffreyjurgajtis**
This PR adds documentation for Email Previews including the following endpoints:

- POST /v1/preview/tests
- GET /v1/preview/tests
- GET /v1/preview/tests/{TEST_ID}
- GET /v1/preview/tests/{TEST_ID}/results
- GET /v1/preview/tests/{TEST_ID}/content
- DELETE /v1/preview/tests/{TEST_ID}
- PUT /v1/preview/tests/{TEST_ID}/results/reprocess
- GET /v1/preview/tests/clients

### CODE REVIEW

File: source/api-email-preview.rst [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+Get Test Info
+-------------
+
+This call returns the subject and submission time in UNIX timestamp format. It will also contain
+one to three properties containing an array of clients. The ``completed`` property shows clients
+that have completed screenshots uploaded. The ``processing`` property contains clients which are
+still being processed by our system. The ``bounced`` property contains clients that were bounced
+by the destination and cannot be retried.
+
+This call will automatically requeue screenshots if they stay in processing for more than threeminutes.
```
Comment (mkbond) [inline source/api-email-preview.rst:102]:
```suggestion
This call will automatically requeue screenshots if they stay in processing for more than three minutes.
```

**Review by mkbond — CHANGES_REQUESTED** [review]
2 comments but the rest LGTM!

**Review by roaet — APPROVED** [review]
⛄

**Comment by roaet** [comment]
Great docs

File: source/api-email-preview.rst [hunk 2]
```diff
@@ -0,0 +1,341 @@
+.. _api-email-preview:
+
+Email Preview
+=============
+
+Our Email Preview service allows you to see how your emails render across various web, desktop, and mobile clients.
+
+Create Test
```
[the remark this answers is not in the document]
Comment (jeffreyjurgajtis) [reply source/api-email-preview.rst:8]: good call! added in https://github.com/mailgun/documentation/pull/582/commits/3bf51ddbc12be414d136f6cf9e84793a0eb20279

File: source/api-email-preview.rst (hunk 1 above)
Comment (jeffreyjurgajtis) [reply source/api-email-preview.rst]: good catch! fixed in https://github.com/mailgun/documentation/pull/582/commits/3bf51ddbc12be414d136f6cf9e84793a0eb20279

**Review by mkbond — APPROVED** [review]

**Comment by Jvb182** [comment]
Baller

**Pull request merged**